demo/src/screens/componentScreens/ButtonsScreen.tsx

```diff
@@ -1,6 +1,6 @@
 import React, {Component} from 'react';
 import {ScrollView, StyleSheet, Alert, Image} from 'react-native';
-import {Text, View, Assets, Constants, Button, Colors, Typography} from 'react-native-ui-lib'; //eslint-disable-line
+import {Text, View, Assets, Constants, Button, Colors, Typography, ButtonProps} from 'react-native-ui-lib'; //eslint-disable-line
```

Comment:
Same here, you can remove the `//eslint-disable-line`

Reply:
Fixed